consoles/sshVirtsh.pm

```diff
@@ -496,6 +499,33 @@ sub resume ($self) {
 
 sub get_remote_vmm ($self) { $bmwqemu::vars{VMWARE_REMOTE_VMM} // '' }
 
+sub _encode_config ($self, $file) {
+    unless (!!$file) {
+        croak "Please provide a configuration file to encode!";
+    }
```

Comment:
```suggestion
    croak 'Please provide a configuration file to encode!' unless $file;
```
(There's no need to use the `!!`-syntax here as `$file` would be converted to a boolean either way.)

Reply:
Fixed. I usually do this in order to make sure that the conversion happens. I find it more readable, but that is a matter of personal taste.

Reply:
So I have tested the code without a parameter, and the function wouldn't be even callable. Hence, I have removed this condition